requirements.txt

```diff
@@ -10,3 +10,4 @@ httpx==0.24.0
 pylint==2.17.4
 langchain==0.0.165
 python-multipart==0.0.6
+supabase==1.0.3
```

Comment:
did you try using httpx==0.23.0? May it will work. I must have pinned it to 0.24.0 only because its the latest.